code/game/objects/items/devices/antivirus.dm

```diff
@@ -9,9 +9,9 @@
 		var/mob/living/carbon/human/H = M
 		var/cured = 0
 		if(MOB_ROBOTIC in H.mob_biotypes)
-			H.say("Installing [src]. Please do not turn your [H.dna.species] unit off or otherwise disturb it during the installation process")
+			H.forcesay("Installing [src]. Please do not turn your [H.dna.species] unit off or otherwise disturb it during the installation process")
```

Comment:
`H.say("Installing [src]. Please do not turn your [H.dna.species] unit off or otherwise disturb it during the installation process", forced = "antivirus")`

This could also work